cypress/integration/pages/mediaAssetPage/helpers.js

```diff
@@ -0,0 +1,16 @@
+export const getBlockByType = (blocks, blockType) => {
+  let blockData;
+
+  blocks.forEach(block => {
+    if (!blockData && block.type === blockType) {
+      blockData = block;
+    }
+  });
+  return blockData;
```

Comment:
Arrays have a `find` method for making this a bit more direct :)

Code could look something like:

```js
return blocks.find(block =>
  block.type === blockType;
);
```

Ref: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find